### PR TITLE
190 refactor adding nodes in semantics module

### LINKED ISSUE #190: Refactor adding nodes in semantics module
## Description
In the semantics module, there is a function of adding nodes. However, currently, it only forwards the request to the entity creation view.

## Suggestion
Prefill a relationship with the node that is selected. In this way, the graph is really extended from the selected node.

### PR DESCRIPTION
**Pull request by sbanoeon**

### CODE REVIEW

**Comment by djs0109** [comment]
closes #190

**Comment by djs0109** [comment]
closes #166

**Review by djs0109 — APPROVED** [review]

File: app/Entirety/entities/views.py [hunk 1]
```diff
@@ -248,23 +264,13 @@ def post(self, request, *args, **kwargs):
                         i = i + 1
                     j = j + 1
                 res = post_entity(self, entity, False, self.project)
-                if res:
```
Comment (djs0109) [inline app/Entirety/entities/views.py:251]: @sbanoeon, this is duplicated, right?

Comment (sbanoeon) [reply app/Entirety/entities/views.py:251]: This part of code is delete.

Comment (djs0109) [reply app/Entirety/entities/views.py:251]: Yes, I deleted it yesterday. I would like to know your opinion

Comment (sbanoeon) [reply app/Entirety/entities/views.py:251]: Oh okay, let me test and have a look at that.

Comment (sbanoeon) [reply app/Entirety/entities/views.py:251]: Yes, indeed, it was duplicated.

**Pull request merged**